api/requirements.txt

```diff
@@ -0,0 +1,3 @@
+redis==2.10.3
```

Comment:
Since this introduces a `requirements.txt` file, I believe it would be best if you sourced these requirements to the `setup.py` file too, in order to keep requirements in a centralized location.

ie, you can change [this line](https://github.com/sourcelair/ceryx/blob/master/api/setup.py#L15) to something like:

```python
install_requires=requirements.read().split('\n')
```